src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/ImageRenderer.js

```diff
@@ -8,6 +8,7 @@ import imageRendererStyles from './imageRenderer.scss';
 import type {Hotspot, Value} from './types';
 import type {IObservableValue} from 'mobx/lib/mobx';
 import type {ElementRef} from 'react';
+import symfonyRouting from 'fos-jsrouting/router';
```

Comment:
Did see the CI also fails because library import need to be beofre other imports:

> error  `fos-jsrouting/router` import should occur before import of `./imageRenderer.scss`  import/order